components/brave_wallet_ui/components/desktop/popup-modals/add-account-modal/hardware-wallet-connect/index.tsx

```diff
@@ -41,13 +41,28 @@ export default function (props: Props) {
     LedgerDerivationPaths.LedgerLive.toString()
   )
 
+  const onSelectedDerivationScheme = (scheme: string) => {
+    setSelectedDerivationScheme(scheme)
+    setAccounts([])
```

Comment:
@Douglashdaniel I think we should not hide account list if accounts are empty, pls check video it automatically shows connect button again

Reply:
https://github.com/brave/brave-browser/issues/18185 as discussed